### PR TITLE
Export/UI: improve "NLA Strips" option name/description

### PR DESCRIPTION
**Pull request by scurest**
(Belated) followup to https://github.com/KhronosGroup/glTF-Blender-IO/pull/764#issuecomment-551403017.

The name and tooltip for the "NLA Strips" option is currently basically impossible to understand unless you already know what it does. We should try to improve it.

How does this look? I tried to make it both descriptive and concise.

Alternative name idea: `Multiple Animations (organized by NLA)`

### CODE REVIEW

**Comment by scurest** [comment]
Alternative text idea: "When on, create a glTF animation named 'Foo' by pushing all actions that should play onto NLA tracks named 'Foo'."

**Comment by scurest** [comment]
cc @donmccurdy

**Comment by julienduroure** [comment]
I like this new description, that reflects really better the feature. If Don validates it too, we need to update the documentation too.

**Comment by donmccurdy** [comment]
Perhaps "Group by" rather than "Organize by"? Otherwise this looks good!

**Comment by scurest** [comment]
I'm good with "Group by". Should I go with that?

**Comment by scurest** [comment]
I changed it to "Group by" and updated the option name in the docs.

The docs about animation (all the stuff about "NLA Strip animations") could probably be reworked too. Basically I wrote those in reverse: given that the "NLA Strips" option meant "Export NLA Strip animations", I tried to write a definition for "NLA Strip animation" that would explain what that meant. But I'm not gonna try to change it in this PR.

**Comment by emackey** [comment]
One suggestion, otherwise looks good to me too.

File: addons/io_scene_gltf2/__init__.py [hunk 1]
```diff
@@ -274,8 +274,12 @@ def __init__(self):
     )
 
     export_nla_strips: BoolProperty(
-        name='NLA Strips',
-        description='Export NLA Strip animations',
+        name='Group by NLA Track',
+        description=(
+            "When on, two actions become part of the same glTF animation if they're\n"
```
[the remark this answers is not in the document]
Comment (scurest) [reply addons/io_scene_gltf2/__init__.py]: Done.

**Pull request merged**